sklearnex/preview/decomposition/pca.py

```diff
@@ -308,7 +299,7 @@ def fit_transform(self, X, y=None):
             Transformed values of X.
         """
         if self.svd_solver in ["randomized", "arpack"]:
-            return sklearn_PCA.fit_transform(self, X)
+            return super().fit_transform(X)

```

Comment:
There is some nasty recursion that can be induced if this change isn't made, unfortunately this is caused by the fit_transform in sklearn using fit() which because we are passing 'self' will lead to this code running again.  It is a temporary fix for the bad design (and is pythonic)